packages/flutter/lib/src/widgets/autocomplete.dart

```diff
@@ -366,10 +384,7 @@ class _RawAutocompleteState<T extends Object> extends State<RawAutocomplete<T>>
 
     // Make sure the options are no longer hidden if the content of the field
     // changes (ignore selection changes).
```

Comment:
This comment is kind of irrelevant now, I think you can just remove it.

Reply:
sure